src/lib/helpers/utils.ts

```diff
@@ -66,7 +68,12 @@ export function jsonStringify(unknown: unknown): string {
 
 export function stringifyUnknown(unknown: unknown): string {
```

Comment:
Could you please add a comment on what's the use-case for this function and what cases this function covers. 

Seems like it's error oriented, maybe call it correspondingly?

Reply:
yeah I mostly use this with errors cause one can throw anything in js (not only errors) and it's good to be able to represent the error in a string form. but I also used it for e.g. debug logs or "unreachable error" messages where we don't neceserraly care much about the output as long as it's somewhat readable. Maybe would be good to have two functions for this each with a more clear purpose for existance and used accordingly. Added task to do that https://linear.app/fluence/issue/DXJ-763/refactor-stringifyunknown-function